CodeEditTests/Features/TerminalEmulator/Shell/ShellTests.swift

```diff
@@ -45,8 +45,14 @@ final class ShellTests: XCTestCase {
 
         // Additional assertion to check output
         let outputData = outputPipe.fileHandleForReading.readDataToEndOfFile()
-        let outputString = String(decoding: outputData, as: UTF8.self).trimmingCharacters(in: .whitespacesAndNewlines)
-        XCTAssertTrue(outputString.contains("Testing"))
+        if let outputString = String(
+            bytes: outputData,
+            encoding: .utf8
+        )?.trimmingCharacters(in: .whitespacesAndNewlines) {
+            XCTAssertTrue(outputString.contains("Testing"))
+        } else {
+            XCTFail("Failed to decode output data")
+        }
```

Comment:
Clearer method to unwrap an optional (will need to mark this function as throwing)
```suggestion
        let outputString = try XCTUnwrap(
            String(bytes: outputData, encoding: .utf8)?.trimmingCharacters(in: .whitespacesAndNewlines)
        )
        XCTAssertTrue(outputString.contains("Testing"))
```

Reply:
That is clearer, done!